puppet/Puppetfile

```diff
@@ -16,10 +16,10 @@ mod "example42/monitor", '2.0.1'
 mod "datagovuk/solr", '2.0.7',
    :git => "https://github.com/example42/puppet-solr"
 
-# commit e92de3dee9c55d5 is actually v1.9.5
+# commit 9ef5e39 is actually v1.9.5
 mod "datagovuk/python",
    :git => "https://github.com/stankevich/puppet-python",
-   :ref => 'e92de3dee9c55d5'
+   :ref => '9ef5e39'
```

Comment:
You're right. WIll fix that. `e92de3dee9c55d5` however is definitely out of the history.